doc/source/advanced/datatype.rst

```diff
@@ -0,0 +1,90 @@
+***********************************
+Data Types in Lux
+***********************************
+
+.. note:: You can follow along this tutorial in a Jupyter notebook. 
+
+Currently, Lux supports four data types:
+
+* Quantitative
+* Temporal
+* Nominal
+* ID
+
+Note that these data types are different from Pandas `dtype`. These data types are automatically inferred based on the `dataframe` inputted.
+The following code snippet shows what data types were inferred:
+
+.. code-block:: python
+
+  df = pd.read_csv("https://github.com/lux-org/lux-datasets/blob/master/data/college.csv?raw=true")
+  df.data_type
+
+
+Quantitative
+------------
+The Quantitative data type is used when there is a count or measure of a certain attribute. 
```

Comment:
I think we should first say that something is quantitative if it is a numerical column. Measure/counts are examples of quantitative but they don't define it.